### PR TITLE
GEODE-52: Remove @author tags from Java source

### PR DESCRIPTION
**Pull request by sboorlagadda**
* removed @author tags except from java source copied from JSON.org
* removed @author from java script files.

I have automatically removed @author tags using unix command ` find . -type f -print0 | xargs -0 sed -i /@author/d`

I tested gradle build to check any compilation issues due to auto-remove a line from headers.

Also precheckin passed.

### CODE REVIEW

**Comment by pdxrunner** [comment]
In some cases, removing the @author tag leaves an empty javadoc. I think it would be better to remove the entire block for these cases.

As an examnple:
diff --git a/gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/BucketRegionQueue.java b/gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/BucketRegionQueue.java
index f7eaf2b..6b76c31 100644
--- a/gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/BucketRegionQueue.java
+++ b/gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/BucketRegionQueue.java
@@ -55,7 +55,6 @@
 import com.gemstone.gemfire.internal.offheap.OffHeapRegionEntryHelper;
 
 /**
- * @author Suranjan Kumar
  * 
  */
 public class BucketRegionQueue extends AbstractBucketRegionQueue {

**Comment by sbawaska** [comment]
+1

**Comment by upthewaterspout** [comment]
+1

I'm curious about why you left the json.org code alone, but not gemfire-joptsimple. Not a big deal because either way I think we have plans to get rid of both of those pieces of copied code.

File: extensions/gemfire-modules-hibernate/src/test/java/com/gemstone/gemfire/modules/Owner.java [hunk 1]
```diff
@@ -34,9 +34,6 @@
 /**
  * Simple JavaBean domain object representing an owner.
  * 
- * @author Ken Krebs
```
Comment (metatype) [inline extensions/gemfire-modules-hibernate/src/test/java/com/gemstone/gemfire/modules/Owner.java]: This files looks to be copied from spring-petclinic.  We need to preserve the author tags and reapply the original copyright header.

File: extensions/gemfire-modules-session/src/test/java/com/gemstone/gemfire/modules/session/junit/NamedRunner.java [hunk 2]
```diff
@@ -27,7 +27,6 @@
  * marked with @After and @Before (but also the Test method itself) can request
  * this name for which it is running.
  *
- * @author Rudy De Busscher
  */
 public class NamedRunner extends BlockJUnit4ClassRunner {
```
Comment (metatype) [inline extensions/gemfire-modules-session/src/test/java/com/gemstone/gemfire/modules/session/junit/NamedRunner.java]: This code appears to be copied from Apache myfaces project.  We should preserve the author tag and possibly add to NOTICE.

File: extensions/gemfire-modules-session/src/test/java/com/gemstone/gemfire/modules/session/junit/SeparateClassloaderTestRunner.java [hunk 3]
```diff
@@ -23,7 +23,6 @@
 import java.net.URLClassLoader;
 
 /**
- * @author StackOverflow
```
Comment (metatype) [inline extensions/gemfire-modules-session/src/test/java/com/gemstone/gemfire/modules/session/junit/SeparateClassloaderTestRunner.java]: We need to investigate the origin of this code to ensure it is safe to include in the geode code base.

**Comment by sboorlagadda** [comment]
Closing this pull-request as this work is re-done due to name change for sub-projects from Gemfire to Geode and the new re-work in on feature/GEODE-52 branch.

**Pull request closed without merging**